extensions/amp-wpm-player/0.1/test/test-amp-wpm-player.js

```diff
@@ -0,0 +1,124 @@
+/**
+* Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS-IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import '../amp-wpm-player';
+
+describes.realWin('amp-wpm-player', {
+  amp: {
+    extensions: ['amp-wpm-player'],
+  },
+  allowExternalResources: true,
+}, env => {
+  const defaultOptions = {
+    layout: 'responsive',
+    width: 1920,
+    height: 1080,
+    id: 'wpmPlayer',
+    url: 'https://wp.tv/?mid=2013067',
+    autoplay: 'true',
+    adv: 'true',
+    extendedrelated: 'false',
+    forceliteembed: 'false',
+  };
+
+  let win;
+  let doc;
+
+  beforeEach(() => {
+    win = env.win;
+    doc = win.document;
+  });
+
```

Comment:
Would be useful to add tests for attribute parsing since it seems to be integral here.